packages/medusa/src/api/routes/admin/products/list-products.ts

```diff
@@ -83,6 +83,16 @@ import { FilterableProductProps } from "../../../../types/product"
  *       type: array
  *       items:
  *         type: string
+ *   - in: query
+ *     name: category_id
+ *     style: form
+ *     explode: false
+ *     description: Category ids to search for.
+ *     schema:
+ *       type: array
+ *       items:
+ *         type: string
+ *   - (query) category_children {boolean} Search for products with category including children.
```

Comment:
suggestion: I would rewrite to something like: "Include category children when filtering by category_id." 
@shahednasser might have a better formulation than I for this.